tikv/raftstore/msg.go

```diff
@@ -0,0 +1,145 @@
+package raftstore
+
+import (
+	"time"
+
+	"github.com/coreos/etcd/raft"
+	"github.com/ngaut/unistore/rocksdb"
+	"github.com/pingcap/kvproto/pkg/import_sstpb"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/kvproto/pkg/raft_cmdpb"
+	"github.com/pingcap/kvproto/pkg/raft_serverpb"
+)
+
+type MsgType int
+
+const (
+	MsgTypeRaftMessage            MsgType = 1 //
```

Comment:
I prefer PeerMsgTypeXXX and StoreMsgTypeXXX.

Reply:
I think `Msg` is the better prefix, it is easier to separate message types from other types that have perfix `Peer` or `Store`.

Reply:
So MsgTypeRaftCmd  should be MsgTypePeerRaftCmd?

Reply:
Message defaults to peer message, a store message has `Store` in the name.

Reply:
OK